clock.py

```diff
@@ -39,7 +39,7 @@ def on_ui_setup(self, ui):
                         memenable = True
                         logging.info(
                             "Pwnagotchi Clock Plugin: memtemp is enabled")
-        if ui.is_waveshare_v2():
+        if ui.is_waveshare_v2() or ui.is_waveshare_v3():
             pos = (130, 80) if memenable else (200, 80)
```

Comment:
```suggestion
            pos = (110, 80) if memenable else (200, 80)
```
You will also need to change this to 110 otherwise if memtemp is enabled it will obscure the clock trough i can't say what change this will bring to users of waveshare v2 because presumatly this worked fine on said versions so maybe it would make more sense to add another check for v2/v3 to differentiate between those two values